public/data/feedData.json

```
@@ -0,0 +1,30 @@
+[
+  {
+    "id": 1,
+    "userName": "wecode",
+    "content": "/images/yonghyun/sky.jpeg",
+    "likeNumber": "좋아요 485,222개",
```

"좋아요" ,"개"는 js에서 작성하고, likeNumber의 value는 숫자로만 주어도 좋을 것 같습니다. ☺️

---

그런 방법이 있었군요....!
수정하겠습니다다다다단님

```suggestion
    "likeNumber": 485222,
```